tests/test_task_runners.py

```diff
@@ -60,9 +60,7 @@ def dask_task_runner_with_existing_cluster(use_hosted_orion):  # noqa
     Generate a dask task runner that's connected to a local cluster
     """
     with distributed.LocalCluster(n_workers=2) as cluster:
-        with distributed.Client(cluster) as client:
-            address = client.scheduler.address
-            yield DaskTaskRunner(address=address)
+        yield DaskTaskRunner(cluster=cluster)
```

Comment:
Can you add an additional fixture that still uses the cluster address so that we can maintain test coverage for that parameter?

Reply:
sure